### ISSUE TITLE
Life events classes should be published

### ISSUE THREAD

**Issue by galund** [alphagov/di-identity-vocab#61]
We aren't currently publishing any of the life events (death registered/registration updated).

We are currently only publishing the JSON Schemas for those events, but not any documentation for them.

This is because they aren't linked from the `credentials.yaml` file.

We can review this once the new layout is in place, but as life events are not an actual feature of the platform at this time, it's definitely not urgent to fix this (and arguably we should hold off doing so, at least until discussion with HMPO or CDDO might agree it's useful).

**Comment by chris-dco** [comment]
This ticket has gone stale. Please open a new ticket if further changes are required.
